desktop/assets/texts/en-US/settings.ftl

```diff
@@ -16,10 +16,10 @@ tcp-connections-allow = Allow
 tcp-connections-ask = Ask
 tcp-connections-deny = Deny
 
-open-url-mode = Open URL Mode
-open-url-mode-allow = Allow
-open-url-mode-confirm = Confirm
-open-url-mode-deny = Deny
+open-links = Open Links
+open-links-allow = Allow
+open-links-ask = Ask
+open-links-deny = Deny
```

Comment:
And here https://github.com/ruffle-rs/ruffle/blob/53199a51164ce338a259f1dba5a850eb7280cf92/desktop/src/gui/open_dialog.rs#L206